### PR TITLE
Products page and Router

### PR DESCRIPTION
**Pull request by varun-Tad**
I have implemented Router and the UI of the products page. Please ignore the content of the cards on the product page. I will change the brand names once UI gets reviewed. You can navigate to the products page by clicking on Shop now button present in the home page.

### Files to review
 1. FilterSection.js
 2.  ProdListingPage.js
 3. ProductsPage.js
 4. ProductsPage.data.js

### CODE REVIEW

**Review by AshutoshM29 — COMMENTED** [review]
Looks good to me, can be merged!!

Things I like about your code:
1. Good use of abbreviations for all files.
2. Proper indentation
3. File structure

File: dressup/src/pages/ProductsPage/FilterSection/FilterSection.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            <input type="checkbox" name="categories" value="Watches"></input>
+            <label htmlFor="Watches"> Watches</label>
+          </li>
+          <li>
+            <input type="checkbox" name="categories" value="Skirts"></input>
+            <label htmlFor="Skirts">Skirts</label>
+          </li>
+          <li>
+            <input type="checkbox" name="categories" value="Shoes"></input>
+            <label htmlFor="Shoes">Shoes</label>
```
Comment (Sahil25061999) [inline dressup/src/pages/ProductsPage/FilterSection/FilterSection.js:66]: just a suggestion you can create checkbox component as it is repeating.

File: dressup/src/pages/ProductsPage/ProdListingPage/ProdListingPage.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import "./ProdListingPage.css";
+import ProdListingCard from "../../../components/Cards/ProdListingCard";
+import { ProductsPageData } from "../ProductsPage.data";
+
+const ProdListingPage = () => {
+  return (
+    <div className="card-vessel">
+      {ProductsPageData.map((item) => {
+        return <ProdListingCard prodData={item}></ProdListingCard>;
+      })}
```
Comment (Sahil25061999) [inline dressup/src/pages/ProductsPage/ProdListingPage/ProdListingPage.js]: provide a unique key for the cards

**Review by Sahil25061999 — APPROVED** [review]
suggested few changes, rest looks good

**Comment by varun-Tad** [comment]
Thank for the review guys. @Sahil25061999 , @AshutoshM29 . Will work on the changes

**Pull request merged**